routes/foodtrucks.go

```diff
@@ -124,7 +124,7 @@ func PostFoodTrucksHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	// Send response
-	w.WriteHeader(http.StatusOK)
+	w.Write([]byte(addedFoodTruck.ID))
```

Comment:
can you keep the StatusOK as well? (before this line)

Reply:
ok